tests/core/test_views_errors.py

```diff
@@ -40,19 +40,16 @@ def test_backoff_header_is_present_on_error_responses(self):
             self.assertEquals(response.headers['Backoff'], '10')
 
     def test_404_is_valid_formatted_error(self):
-        response = self.app.get('/unknown', status=404)
-        self.assertFormattedError(response, 404, ERRORS.MISSING_RESOURCE, "Not Found",
-                                  "The resource you are looking for could not be found.")
-
-    def test_404_can_be_overridded(self):
         custom_404 = http_error(httpexceptions.HTTPNotFound(),
                                 errno=ERRORS.MISSING_RESOURCE,
-                                message="Customized.")
+                                message="Customized.",
+                                details={"id": "abc", "resource_name": "collection"})
         with mock.patch('tests.core.testapp.views.Mushroom._extract_filters',
                         side_effect=custom_404):
             response = self.app.get(self.sample_url, headers=self.headers, status=404)
         self.assertFormattedError(
-            response, 404, ERRORS.MISSING_RESOURCE, "Not Found", "Customized.")
+            response=response, code=404, errno=ERRORS.MISSING_RESOURCE, error="Not Found",
+            message="Customized.", details={"id": "abc", "resource_name": "collection"})
```

Comment:
I found tests for unknown records in `tests/core/resource/test_record.py`.
I couldn't find anything `relevant in tests_views_*.py`
Am I missing something?